frontend/cypress/integration/featureFiles/overview.feature

```diff
@@ -101,12 +101,6 @@ Feature: Kiali Overview page
     Then there should be a "degraded" application indicator in the namespace
     And the "degraded" application indicator should list the application
 
-  @error-rates-app
```

Comment:
Yes, it will need to migrate.  But I needed to remove it here because the information is no longer on the overview card, it is moved to the control plane node side panel.